### ISSUE TITLE
All episodes switch (-A) doesn't work for SVTPLAY

### ISSUE THREAD

**Issue by keyboardsmash** [spaam/svtplay-dl#1276]
### svtplay-dl versions:
svtplay-dl 2.7

### Operating system and Python version:
MacOS 10.14.6 (18G3020) 
Python 2.7.16

Linux Ubuntu 20.04.1 LTS
Python 3.8.5

### What is the issue:
All episodes switch (-A) makes svtplay-dl crash

LINUX:

```
 me@linux:/raid/SVT_new$ svtplay-dl -A --verbose https://www.svtplay.se/video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden
DEBUG [1604869656.3598225] /usr/lib/python3/dist-packages/svtplay_dl/utils/getmedia.py/get_media: version: 2.7
DEBUG [1604869656.3632467] /usr/lib/python3/dist-packages/svtplay_dl/service/__init__.py/__init__: service: svtplay
DEBUG [1604869656.3638453] /usr/lib/python3/dist-packages/svtplay_dl/utils/http.py/request: HTTP getting 'https://www.svtplay.se/video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden'
DEBUG [1604869656.3662035] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): www.svtplay.se:443
DEBUG [1604869656.4315536] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: https://www.svtplay.se:443 "GET /video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden HTTP/1.1" 200 50854
Traceback (most recent call last):
  File "/usr/bin/svtplay-dl", line 7, in <module>
    svtplay_dl.main()
  File "/usr/lib/python3/dist-packages/svtplay_dl/__init__.py", line 66, in main
    get_media(urls[0], config, __version__)
  File "/usr/lib/python3/dist-packages/svtplay_dl/utils/getmedia.py", line 59, in get_media
    get_all_episodes(stream, url, options)
  File "/usr/lib/python3/dist-packages/svtplay_dl/utils/getmedia.py", line 77, in get_all_episodes
    episodes = stream.find_all_episodes(stream.config)
  File "/usr/lib/python3/dist-packages/svtplay_dl/service/svtplay.py", line 220, in find_all_episodes
    for n in janson[i]['items({"filter":{"includeFullOppetArkiv":false}})']:
KeyError: 'items({"filter":{"includeFullOppetArkiv":false}})'
```

MacOS:
```
➜  ~ svtplay-dl -A --verbose https://www.svtplay.se/video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden
DEBUG [1604869710.167444] /usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/svtplay_dl/utils/getmedia.py/get_media: version: 2.7
DEBUG [1604869710.170088] /usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/svtplay_dl/service/__init__.py/__init__: service: svtplay
DEBUG [1604869710.170662] /usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/svtplay_dl/utils/http.py/request: HTTP getting 'https://www.svtplay.se/video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden'
DEBUG [1604869710.1782458] /usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): www.svtplay.se:443
DEBUG [1604869710.211508] /usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/urllib3/connectionpool.py/_make_request: https://www.svtplay.se:443 "GET /video/15367465/var-tid-ar-nu/var-tid-ar-nu-sasong-1-freden HTTP/1.1" 200 50854
Traceback (most recent call last):
  File "/usr/local/bin/svtplay-dl", line 7, in <module>
    svtplay_dl.main()
  File "/usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/svtplay_dl/__init__.py", line 66, in main
    get_media(urls[0], config, __version__)
  File "/usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/svtplay_dl/utils/getmedia.py", line 59, in get_media
    get_all_episodes(stream, url, options)
  File "/usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/svtplay_dl/utils/getmedia.py", line 77, in get_all_episodes
    episodes = stream.find_all_episodes(stream.config)
  File "/usr/local/Cellar/svtplay-dl/2.7/libexec/lib/python3.9/site-packages/svtplay_dl/service/svtplay.py", line 220, in find_all_episodes
    for n in janson[i]['items({"filter":{"includeFullOppetArkiv":false}})']:
KeyError: 'items({"filter":{"includeFullOppetArkiv":false}})'
➜  ~
```

**Comment by pythonuser3856** [comment]
Please see closed item #1274

**Comment by spaam** [comment]
2.8 have a fix for this.  :)
